lib/HLSL/ComputeViewIdStateBuilder.cpp

```diff
@@ -111,12 +111,13 @@ class DxilViewIdStateBuilder {
     // Contributing instructions per output.
     std::unordered_map<unsigned, InstructionSetType>
         ContributingInstructions[kNumStreams];
-
+    bool IsForPatchConstantOrPrimitive;
```

Comment:
I don't understand why this needs to be a member of EntryInfo.  It's only checked in CollectValuesContributingToOutputs, and the caller of that function knows whether or not it is for the patch constant or primitive.

Reply:
Switched to send it to CollectValuesContributingToOutputs.